src/init.cpp

```diff
@@ -800,6 +831,63 @@ void InitLogging()
  */
 bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
 {
+    // ********************************************************* Step 0: download bootstrap
+    std::string sBootstrap = GetArg("-bootstrap","");
+
+    if(sBootstrap != "")
+    {
+        RemoveConfigFile("bootstrap");
+        uiInterface.InitMessage(strprintf("[BOOTSTRAP] Trying to download from %s", sBootstrap.c_str()));
+        CURL *curl;
+        FILE *fp, *a;
+        CURLcode res;
+
+        curl = curl_easy_init();
+        if (curl) {
```

Comment:
is curl os specific. e.g you need it installed on your machine, or is packaged in the app? - just asking

Reply:
the library is linked, curl is already used in other parts of navcoin-core

Reply:
thanks :)